### PR TITLE
Replaced Demisto name

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/45674)

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/DemistoLocking/Integrations/CoreLock/DemistoLock.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+    case 'demisto-lock-release-all':
+        setVersionedIntegrationContext({}, sync);
+
+        var md = '### Demisto Locking Mechanism\n';
+        md += 'All locks released successfully';
+        return { ContentsFormat: formats.markdown, Type: entryTypes.note, Contents: md } ;
+
+    case 'demisto-lock-info':
+        integrationContext = getVersionedIntegrationContext(sync);
```
Comment (anara123) [inline Packs/DemistoLocking/Integrations/CoreLock/DemistoLock.js:97]: this integration is already packagified. 
Merge from master

File: Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI.yml [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    arguments:
+      - name: uri
+        required: true
+        description: Request URI
+      - name: filename
+        description: File name of download
+      - name: description
+        description: Description of file entry
+    description: Download files from Demisto server
+  - name: core-api-multipart
```
Comment (anara123) [inline Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI.yml:139]: Merge from master.

**Comment by ShahafBenYakir** [comment]
@dansterenson @yaakovi @anara123 Is this waiting for another issue?

**Comment by dansterenson** [comment]
@ShahafBenYakir  I think it is waiting for @anara123 to review the changes.

**Review by anara123 — CHANGES_REQUESTED** [review]

File: Packs/CommonScripts/Scripts/ProvidesCommand/ProvidesCommand.yml [hunk 3]
```diff
@@ -23,10 +23,10 @@ args:
 scripttarget: 0
 dependson:
   must:
-  - Demisto REST API|||demisto-api-post
-  - Demisto REST API|||demisto-api-get
+  - demisto-api-post
+  - demisto-api-get
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply Packs/CommonScripts/Scripts/ProvidesCommand/ProvidesCommand.yml:27]: The script is used both for XSOAR and XSIAM and implemented in both of them. It should work in both of them, I just deleted the "using" integration.

image

**Review by anara123 — APPROVED** [review]

**Pull request merged**